src/cfe.c

```diff
@@ -491,6 +503,36 @@ else
   *surface_runoff_depth_m = 0.0;
   *infiltration_depth_m = 0.0;
   }
+
+
+ // impermeable fraction due to frozen soil
+ double factor = 1;
+ double smc_ref = 0.249; // taken from noahmp soil_params file for sandy loam
+ double frzk = 0.15; // taken from noahmp GENPARM.TBL

```

Comment:
Should these be variable inputs?

Reply:
To piggyback on this comment, are these vars getting set every time the `Schaake_partitioning_scheme` is being called? Also, is `smc_ref` always 0.249? Should it be a parameter instead?

Reply:
okay, so smc_ref is field capacity which is already in cfe, I passed it to the Schaake scheme. The frzk "ice_content above which soil is impermeable" is an input parameter now.